Resources/public/js/models/ez-locationmodel.js

```diff
@@ -178,6 +178,66 @@ YUI.add('ez-locationmodel', function (Y) {
                 location.reset();
                 callback(error, response);
             });
+        },
+
+        /**
+         * Loads path of the current location. The result is the array containing
+         * eZ.Location objects present on the path. The array doesn't contain current location.
+         *
+         * @param {Object} options
+         * @param {Object} options.api the JS REST client instance
+         * @param {Function} callback
+         */
+        loadPath: function (options, callback) {
+            var locations = [],
+                that = this;
+
+            this._loadAncestors(options, function (error, response) {
+                var hits;
+
+                if (error) {
+                    callback(error);
+                    return;
+                }
+
+                hits = response.document.View.Result.searchHits.searchHit;
+
+                Y.Array.each(hits, function (hit) {
+                    var location = new Y.eZ.Location();
+
+                    location.setAttrs(location.parse({document: hit.value}));
+
+                    if (location.get('id') !== that.get('id')) {
```

Comment:
nitpicl: and instead of using `that`, you can set the context of Y.Array.each function with its 3rd argument: http://yuilibrary.com/yui/docs/api/classes/Array.html#method_each